flow/src/main/scala/org/alephium/flow/network/broker/BrokerHandler.scala

```diff
@@ -193,12 +164,10 @@ trait BrokerHandler extends FlowDataHandler {
 
   @SuppressWarnings(Array("org.wartremover.warts.IsInstanceOf"))
   def flowEvents: Receive = {
-    case BlockChainHandler.BlockAdded(hash) =>
-      blockFlowSynchronizer ! BlockFlowSynchronizer.BlockFinalized(hash)
+    case _: BlockChainHandler.BlockAdded => ()
```

Comment:
This is not covered by test. Is it not needed any more?

Reply:
Yes, it’s no longer needed. The previous message flow was: `ChainHandler` sent messages to `BrokerHandler`, then `BrokerHandler` forwarded them to `BlockFlowSynchronizer`.

In the latest code, `BlockFlowSynchronizer` subscribes to `FlowDataEvent` messages and handles them separately in V1 and V2 handlers. I'll add more tests in `BrokerHandler`.